src/components/header/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Menu } from '../';
+import './style.css';
+
+const mainMenu = [
+    { text: 'New',    url: 'https://news.ycombinator.com/newest' },
+    { text: 'Show',   url: 'https://news.ycombinator.com/show' },
+    { text: 'Submit', url: 'https://news.ycombinator.com/submit' },
+];
+
+const numbers = [1 ,2 ,3];
+const double = a => a * 2;
+console.log(numbers.map(double));
+
+const first = mainMenu[0];
+console.log(first.text);
+console.log(first.url);
```

Comment:
line from 11 to 17 was just a demo by me to illustrate how `Array.map` works and how to access object keys.

you don't need it anymore